### PR TITLE
Link dev

### PR DESCRIPTION
**Pull request by noah-puckett**

### CODE REVIEW

File: src/make-table-row.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// //ok so this function creates a ROW
+// //that looks like this:
+
```
Comment (Zilula) [inline src/make-table-row.js]: If you are not using the code, delete it from your repo. It adds clutter and can make things more confusing in a production setting.

File: src/roundtripDog.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        const json = JSON.stringify(dogs);
+        //RIGHT TO LEFT: ADD our KEY VALUE PAIR ('dogs'--key, json--value) to our OBJECT (roundtripdog)'s STORAGE
+        roundtripDog.storage.setItem('dogs', json);
+        
+    },
+    
+    // getMethod() {
+    //     //RIGHT TO LEFT: store our ARRAY OBJECT and save it in a variable (dogs)?
+    //     const dogs = roundtripDog.getAllMethod();
+    //     //RIGHT TO LEFT: give JSON's parse METHOD localStorage's STRING (json) and SAVE it in a VARIABLE (dog)
```
Comment (Zilula) [inline src/roundtripDog.js]: Again unused code should be deleted.

File: src/make-table-row.js (hunk 1 above)
Comment (noah-puckett) [reply src/make-table-row.js]: Awesome catch, thank you so much! Fixed! :)

File: src/roundtripDog.js (hunk 2 above)
Comment (noah-puckett) [reply src/roundtripDog.js]: Fixed!

**Pull request merged**